src/main/java/com/study/battleq/modules/board/controller/BoardApiController.java

```diff
@@ -0,0 +1,56 @@
+package com.study.battleq.modules.board.controller;
+
+import com.study.battleq.infrastructure.common.dto.ResponseDto;
+import com.study.battleq.modules.board.controller.request.DeleteSearchRequest;
+import com.study.battleq.modules.board.controller.request.UpdateBoardRequest;
+import com.study.battleq.modules.board.service.dto.response.BoardSearchResponse;
+import com.study.battleq.modules.board.controller.request.CreateBoardRequest;
+import com.study.battleq.modules.board.service.BoardService;
+import com.study.battleq.modules.board.domain.entity.BoardEntity;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.*;
+
+import javax.validation.Valid;
+import java.util.List;
+
+@RestController
+@RequiredArgsConstructor
+public class BoardApiController {
+
+    private final BoardService boardService;
+
+    @PostMapping("/api/v1/boards")
+    public ResponseDto<Long> saveBoard(@RequestBody @Valid CreateBoardRequest request) {
+        Long id = boardService.save(request.toDto());
+        return ResponseDto.ok(id);
+    }
+
+    @GetMapping("/api/v1/boards")
+    public ResponseDto<List> findAllBoard(){
+        List<BoardSearchResponse> boardSearchResponseList = boardService.findAll();
+        return ResponseDto.ok(boardSearchResponseList);
+    }
+
+    @GetMapping("/api/v1/boards/{id}")
+    public ResponseDto<BoardSearchResponse> findById(@PathVariable("id") Long boardId){
+        BoardSearchResponse boardSearchResponse = boardService.findById(boardId);
+        return ResponseDto.ok(boardSearchResponse);
+    }
+
+    @GetMapping("/api/v1/boards/title/{title}")
+    public ResponseDto<List> findByTitle(@PathVariable("title") String title){
+        List<BoardSearchResponse> boardSearchResponseList = boardService.findByTitle(title);
+        return ResponseDto.ok(boardSearchResponseList);
+    }
+
+    @PostMapping("/api/v1/boards/update/{id}")
```

Comment:
RestAPI에 행위가 포함되면 안됩니다. Put이나 PATCH를 사용하는 건 어떨까요? 글씨가 깨지네요 ㅠ
```suggestion
    @PutMapping("/api/v1/boards/{id}")
```